pong.asm

```diff
@@ -21,435 +21,286 @@
 NewGame:
 	jal ClearBoard
 
-	leftLines:
-		li $a0, 10 #the x starting coordinate
-		li $a1, 13 #the y coordinate
-		lw $a2, colorTwo #the color
-		li $a3, 18 #the x ending coordinate
+	Lines:
+		li $a0, 10 
```

Comment:
@BillyBarbaro Yeah I just looked at the file by itself and I agree with you. I like it. Argument revoked.